extensions/wikia/FandomComMigration/FandomComMigration.i18n.php

```diff
@@ -2,73 +2,6 @@
-);
-
-$messages['pl'] = array(
-	'fandom-com-migration-after' => 'Uwaga: Dokonaliśmy migracji domeny Waszej społeczności na fandom.com. Więcej informacji można znaleźć na [https://spolecznosc.wikia.com/wiki/Pomoc:Migracja_domen_Fandomu tej stronie pomocy].',
-	'fandom-com-migration-before' => 'Uwaga: W ciągu następnych kilku tygodni dokonamy migracji domeny Waszej społeczności na fandom.com. Więcej informacji można znaleźć na [https://spolecznosc.wikia.com/wiki/Pomoc:Migracja_domen_Fandomu tej stronie pomocy].',
-);
-
-$messages['pt'] = array(
-	'fandom-com-migration-after' => 'Atenção: nós migramos o domínio de sua comunidade para fandom.com. Descubra mais sobre isso [https://comunidade.wikia.com/wiki/Ajuda:Migra%C3%A7%C3%A3o_de_Dom%C3%ADnio_do_Fandom nessa página de ajuda].',
-	'fandom-com-migration-before' => 'Atenção: nas próximas semanas, estaremos migrando o domínio de sua comunidade para fandom.com. Descubra mais sobre isso [https://comunidade.wikia.com/wiki/Ajuda:Migra%C3%A7%C3%A3o_de_Dom%C3%ADnio_do_Fandom nessa página de ajuda Fandom].',
-);
-
-$messages['ru'] = array(
-	'fandom-com-migration-after' => 'Внимание. Домен вашего сообщества был изменён с wikia.com на fandom.com. Подробнее см. [http://ru.community.wikia.com/wiki/%D0%A1%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D0%B0:%D0%97%D0%B0%D0%BC%D0%B5%D0%BD%D0%B0_wikia.com_%D0%BD%D0%B0_fandom.com_%D0%B2_URL_%D0%B2%D0%B8%D0%BA%D0%B8%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%BE%D0%B2 справочную страницу].',
-	'fandom-com-migration-before' => 'Внимание. В течение следующих недель домен вашего сообщества изменится с wikia.com на fandom.com. Подробнее см. [http://ru.community.wikia.com/wiki/%D0%A1%D0%BF%D1%80%D0%B0%D0%B2%D0%BA%D0%B0:%D0%97%D0%B0%D0%BC%D0%B5%D0%BD%D0%B0_wikia.com_%D0%BD%D0%B0_fandom.com_%D0%B2_URL_%D0%B2%D0%B8%D0%BA%D0%B8%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%BE%D0%B2 справочную страницу].',
-);
-
-$messages['zh-hans'] = array(
-	'fandom-com-migration-after' => '请注意：我们已经将您的社区域名迁移到了fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
-	'fandom-com-migration-before' => '请注意：在接下来的几个星期内，我们将会把您的社区域名迁移到fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
-);
-
-$messages['zh-hant'] = array(
-	'fandom-com-migration-after' => '请注意：我们已经将您的社区域名迁移到了fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
-	'fandom-com-migration-before' => '请注意：在接下来的几个星期内，我们将会把您的社区域名迁移到fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
+	'fandom-com-migration-before' => "Attention: Over the next few weeks, we are migrating your community to the Unified Community Platform. Find out more on [https://community.fandom.com/wiki/Help:Unified_Community_Platform] this Fandom help page.",
```

Comment:
Should we reuse the same message? something ucp related would be more appropriate. 
I'm not an expert in i18, but when reusing the message don't we risk downloading translations for the previous string?